database/postgres.go

```diff
@@ -10,11 +10,12 @@ import (
 	"strings"
 	"time"
 
-	"codeberg.org/emersion/soju/xirc"
 	_ "github.com/lib/pq"
 	"github.com/prometheus/client_golang/prometheus"
 	promcollectors "github.com/prometheus/client_golang/prometheus/collectors"
 	"gopkg.in/irc.v4"
+
+	"codeberg.org/emersion/soju/xirc"
```

Comment:
Sorry for these noop changes. My formatter really wants to put the package import at the bottom. These can be removed before merging.